pep-3157.txt

```diff
@@ -0,0 +1,282 @@
+PEP: 3157
+Title: Syntactic formalization of f-strings
+Version: $Revision$
+Last-Modified: $Date$
+Author: Pablo Galindo <pablogsal@python.org>,
+        Batuhan Taskaya <batuhan@python.org>,
+        Lysandros Nikolaou <lisandrosnik@gmail.com>
+Sponsor: Pablo Galindo <pablogsal@python.org>
+Discussions-To: python-dev@python.org
```

Comment:
Nit: maybe have discussions on Discourse instead? What have you been doing with your latest PEPs?

Reply:
Ah apologies, yeah discussions are going 100% to discourse 👍